site/content/tutorial/04-logic/05-keyed-each-blocks/text.md

```diff
@@ -2,18 +2,20 @@
 title: Keyed each blocks
 ---
 
-By default, when you modify the value of an `each` block, it will add and remove items at the *end* of the block, and update any values that have changed. That might not be what you want.
+By default, when you add / remove an item from the iterable that the `each` block iterates over, it will add / remove the component instance at the *end* of the block instead of the instance corresponding to that element of the iterable. If you add / remove an item from anywhere other than the end of the iterable, this leads to each component instance corresponding to a different element of the iterable than the one it was first created for. That might not be what you want.
```

Comment:
> ... instead of the instance corresponding to that element of the iterable.

hmm.. this makes me even more confused

Reply:
@tanhauhau You're right, this doesn't sound right. I think I was still too confused while I wrote this. I'll simplify it more.

I just noticed that I can't edit this PR anymore because I deleted my fork. 🤦‍♂️ I'll close this and open a new one if that's ok with you.